Okta.AspNet.Abstractions/StrictSecurityTokenValidator.cs

```diff
@@ -0,0 +1,35 @@
+﻿// <copyright file="StrictSecurityTokenValidator.cs" company="Okta, Inc">
+// Copyright (c) 2018-present Okta, Inc. All rights reserved.
+// Licensed under the Apache 2.0 license. See the LICENSE file in the project root for full license information.
+// </copyright>
+
+using System;
+using System.IdentityModel.Tokens.Jwt;
+using System.Security.Claims;
+using Microsoft.IdentityModel.Tokens;
+
+namespace Okta.AspNet.Abstractions
+{
+    public sealed class StrictSecurityTokenValidator : ISecurityTokenValidator
+    {
+        private readonly JwtSecurityTokenHandler _handler;
+
+        public StrictSecurityTokenValidator(OktaWebOptions options)
+        {
+            _handler = new StrictTokenHandler();
+        }
+
+        public bool CanValidateToken => _handler.CanValidateToken;
+
+        public int MaximumTokenSizeInBytes
+        {
+            get => _handler.MaximumTokenSizeInBytes;
+            set => throw new NotImplementedException();
```

Comment:
I'm wondering if we should throw something other than `NotImplementedException`. Usually when I see that I see it as either a bug in the library that threw it, or an indication that a feature is only partially implemented. But in our case the code is complete, it's just that the property is readonly.

Maybe `NotSupportedException`? Up to you.